rollup.config.js

```diff
@@ -43,7 +43,7 @@ if (BUNDLE) {
 
 plugins.push(minify());
 
-const rollupConfig = [
+let rollupConfig = [
```

Comment:
don't think this needed to be a `let` ?

Reply:
No this was due to me ovveriding locally but have removed that now.